example/lib/main.dart

```diff
@@ -38,15 +38,15 @@ class MainApp extends StatelessWidget {
             ],
           ),
         ),
-        isTopBarLayerAlwaysVisible: true,
         topBarTitle: Text('Pagination', style: Theme.of(context).textTheme.titleSmall),
+        isTopBarLayerAlwaysVisible: true,
         trailingNavBarWidget: IconButton(
           padding: const EdgeInsets.all(16),
           icon: const Icon(Icons.close),
           onPressed: Navigator.of(modalSheetContext).pop,
         ),
         child: const Padding(
-            padding: EdgeInsets.only(bottom: 150),
+            padding: EdgeInsets.fromLTRB(16, 16, 16, 150),
```

Comment:
We should put all hard-coded constants at the top of the file with readable names

Reply:
done in here: d823b9c26ddafe2564c029cd7cec5142b0564b49